src/app/Instance/ErrorHandling/ErrorHandlingDetail.tsx

```diff
@@ -64,36 +78,74 @@ export const ErrorHandlingDetail = ({
   }, [errorHandlingParameters]);
 
   return (
-    <DescriptionList>
-      <DescriptionListGroup
-        data-ouia-component-type="ProcessorConfig/FormGroup"
-        data-ouia-component-id="error_handling_method"
-        key="error-handling-method"
-      >
-        <DescriptionListTerm data-testid="error_handling_method">
-          {t("common.errorHandlingMethod")}
-        </DescriptionListTerm>
-        <DescriptionListDescription data-testid="error_handling_method-value">
-          {errorHandlingMethodLabel}
-        </DescriptionListDescription>
-      </DescriptionListGroup>
-      {errorHandlingParameters && schema && !isSchemaLoading && !apiError && (
-        <ConfigParameters
-          schema={schema}
-          parameters={errorHandlingParameters as { [key: string]: unknown }}
-        />
-      )}
-      {isSchemaLoading && errorHandlingParametersSkeleton}
-      {apiError && (
-        <Alert
-          className="instance-page__tabs-error-handling__alert"
-          ouiaId="error-schema"
-          variant="danger"
-          title={apiError}
-          aria-live="polite"
-          isInline
-        />
-      )}
-    </DescriptionList>
+    <PageSection
+      variant={PageSectionVariants.light}
+      isFilled
+      className={"pf-u-h-100"}
```

Comment:
Your comment made me think. I guess we could also improve things a bit more.

For the new layout to work, all the page sections inside the error handling area must have both `isFilled` and `height: 100%`. Changing this configuration in any of them will break something. I think this is not so explicit when looking at the code cause it's split across different files.

I moved these page components to a separate one (`ErrorHandlingPageSection`) to make things easier to understand.

About having an enum to store the value of `pf-u-h-100`, I think it's not so worth. It doesn't guarantee that the class will always exist (it's a CSS utility class [provided by PF](https://www.patternfly.org/v4/utilities/sizing#usage-1)) nor that it will do its work. It will help making sure we write it right, but at the cost of remembering how the enum is named. I think it's easier to just refer to PF docs when in need.